### ISSUE TITLE
Email Notifications created in Staging toolkit are not sending email to the panellists.

### ISSUE THREAD

**Issue by onepointpb** [OnePointGlobal/OnePoint-Global-Account-Toolkit#306]
But it is working in Live environment.

**Comment by onepointpb** [comment]
I created two more Email Notifications. The third one is now working. All are created in the same manner. Need to check what is the issue with the other two.

**Comment by vivek-onepoint** [comment]
Hi,
I have checked the Email Notification with all the filter conditions in Toolkit. It is working perfectly fine.
Can u please share me the scenario in which it failed, so i can debug and let u know the issue if any.

**Comment by onepointpb** [comment]
1. I have created two more surveys (ids=153020, 153021) and added scheduler then notification by following the below scenario.
**Scenario:** 

- Create a schedule. Don't use previously saved email template. Use "OnePoint" as email server. 
- Create a notification. Don't use previously saved email template. Use "OnePoint" as email server. 

2. For these two surveys I did not received any invite. The JSON is showing no Email server attached to it where as the front end is showing the email server attached.
3. For the above two surveys I did not get any notification. The JSON is showing no Email server attached. The front end also showing no email server that was attached before.
4. After a delay I started receiving the notification for the second survey (id = 153021), even though the JSON and front end both showing no email server.
5. There was no start and end notification as well.

**Comment by vivek-onepoint** [comment]
Hi,
I have created a survey (153019) and send the notification as u have mentioned the scenario. I got the notification Email. 
I have create one more survey (153023) and send schedule Invitation I m got the invitation email.
Delay in getting notification can be due to time difference in servers or can be because of smtp (as it stores email id's in queue to process)
For the same survey i got start and end Email Notification for that survey too.
In JSON Content if EmailServer = 0 that means it will send email through default email Server.
